src/diffusers/utils/testing_utils.py

```diff
@@ -259,6 +259,23 @@ def require_peft_backend(test_case):
     return unittest.skipUnless(USE_PEFT_BACKEND, "test requires PEFT backend")(test_case)
 
 
+def require_peft_version(peft_version):
+    """
+    Decorator marking a test that requires PEFT backend with a specific version, this would require some specific
+    versions of PEFT and transformers.
+    """
+
+    def decorator(test_case):
+        correct_peft_version = is_peft_available() and version.parse(
+            version.parse(importlib.metadata.version("peft")).base_version
+        ) > version.parse(peft_version)
```

Comment:
So when I pass `peft_version="0.6.2"`, this means the version has to be 0.6.3 or higher because of `>`? I think it should be `>=`. If that's so, the error message below should also be adjusted to "greater or equal". 

If strictly greater is actually the desired functionality, I would rename the function, but I think that greater or equal is what you'd normally expect from such a check.

Reply:
I agree it is confusing, changed it to `require_peft_version_greater`

Reply:
Personally, I would find `>=` more intuitive, as this allows to indicate the min version, but I guess this is also fine.

Reply:
Can we separate the conditions storing them into separate vars and then collate them later? This way, the readability gets better. But no strong opinion.